src/java/org/streamingpool/core/domain/DependencyGraph.java

```diff
@@ -0,0 +1,9 @@
+package org.streamingpool.core.domain;
+
+import org.streamingpool.core.service.StreamId;
+
+import java.util.Set;
+
+public interface DependencyGraph {
```

Comment:
Interface Javadoc missing ? Same for public methods in the implementation

Reply:
Thanks for the feedback :)